docs/src/SUMMARY.md

```diff
@@ -4,6 +4,7 @@
 
 # Tutorials
 
+- [Entry-level Model Checker Tutorial](./tutorials/entry-tutorial.md)
 - [Type Checker Tutorial](./tutorials/snowcat-tutorial.md)
```

Comment:
This apparently was introduced previously, but we've ended up with duplicate tutorial listings the TOC. see line 52 below. Could you consolidate them?

Reply:
Yes, merged.